playground/1st-release/public/game.js

```diff
@@ -90,7 +90,7 @@ export default function createGame() {
 
     function movePlayer(command) {
         notifyAll(command)
-
+        
```

Comment:
Nada de mais aqui, mas também se for fazer o uncheck do package.json sugiro fazer desse aqui também :)